### ISSUE TITLE
Add option to "Start With Minimal UI"

### ISSUE THREAD

**Issue by Toneking77** [till213/SkyDolly#156]
**Is your feature request related to a problem? Please describe.**
Not really a problem... just a nice to have for convenience. I have Sky Dolly setup to start up every time I run MSFS.  But then I immediately select the Minimal UI feature instead of minimizing to the Taskbar, because then I forget to start recording (out of sight, out of mind).  With Minimal UI, I see it on my second screen with the reduced footprint along with Little Nav Map, Active Sky, etc.

**Describe the solution you'd like**
Have an option in settings to "**Start With Minimal UI**".

**Describe alternatives you've considered**
An "auto-pause" feature that would allow clicking Record as soon as Sky Dolly loads but doesn't start recording until some condition(s) are met.  Like ground speed increasing or throttle advanced past 30%, something like that. 

**Additional context**
Thank you for creating, maintaining and enhancing Sky Dolly.  Amazing!

**Comment by Toneking77** [comment]
And... I just realized that the Minimal UI state is saved when the program is shut down.  Sorry about that!

**Comment by till213** [comment]
> And... I just realized that the Minimal UI state is saved when the program is shut down. Sorry about that!

Indeed: the window layout (including the table / column layouts) is supposed to be stored in the application settings, including the "minimal UI" state. If that does not work (e.g. also in the context of a "two screen setup") then this would be a bug.

> An "auto-pause" feature that would allow clicking Record as soon as Sky Dolly loads but doesn't start recording until some condition(s) are met. Like ground speed increasing or throttle advanced past 30%, something like that.

I can confirm that such an "auto-record" feature is indeed on my own to-do list since a long time. I took the liberty to rename this issue and will use it once this feature will be implemented.

But to set expectations right: this is somewhat lower in the "backlog", as other (existing) features are to be enhanced / improved first.

**Comment by till213** [comment]
From #157:

Please add an option to automate the start of the recording because many times players forget to record before landing. For example:

Start Recording when I am (xxx) miles from landing

Or

Start Record When I am (x) minutes before the destination

**Comment by stevealexandre** [comment]
> From [#157](https://github.com/till213/SkyDolly/issues/157):
> 
> Please add an option to automate the start of the recording because many times players forget to record before landing. For example:
> 
> Start Recording when I am (xxx) miles from landing
> 
> Or
> 
> Start Record When I am (x) minutes before the destination

Would love to have the auto start record option. Program is in background and it's easy to forget to start. An auto record passing FL or at x MIN would be really appreciated and improve ux. 🙏
